css/index.css.css

```
@@ -0,0 +1,38 @@
+body {
+    background-color: #b6c3d7;
+    color:antiquewhite
```

You're missing a `;` here, which makes the browser identify the `color` and `font`properties as one, with an invalid value.

```suggestion
    color:antiquewhite;
```